README.pipeline.md

```diff
@@ -6,11 +6,25 @@ Actions are defined in `config.yml` file in the task directory.
 
 <a href="https://github.com/mrlvsb/kelvin/blob/master/evaluator/pipelines.py">Builtin actions</a> are implemented directly in Kelvin source code, but it is also possible to run the action in <a href="https://github.com/mrlvsb/kelvin/tree/master/evaluator/images">docker container</a> with concrete compilers, so they don't need to be installed on the server.
 
+If you add a new Docker pipeline, don't forget to also modify the config validation rules
+in `frontend/src/PipelineValidation.js` (variable `rules`).
+
+### The sandbox environment limits
+Submits are evaluated in the sandboxed environment with the help of <a href="https://github.com/ioi/isolate">isolate tool</a>.
```

Comment:
I think we are not using isolate anymore? This could be removed, but limits are still configured with ulimits through docker

Reply:
I changed the remaining mentions to Docker.